bin/demultiplexing.sh

```diff
@@ -173,7 +173,8 @@ log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "Log files will be written
 # Looping through sub dirs to see if all files.
 #
 log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "ls -1 -d ${SEQ_DIR}/*/"
-for i in $(ls -1 -d "${SEQ_DIR}/"*/)
+projects=$(find "${SEQ_DIR}/" -type d)
```

Comment:
De oude ```ls``` pikte alleen directe subdirs op, maar de nieuwe code met ```find``` werkt standaard recursief en zou wel eens iets meer dieper geneste subdirs kunnen oppikken. In de repo zitten andere voorbeelden van find met ``` -mindepth 1 -maxdepth 1``` om het niet recursief te maken ...